apis.js

```diff
@@ -16,6 +16,13 @@ export const fetchArticlesById = (article_id) => {
   });
 };
 
+export const fetchArticlesByTopic = (topic) => {
+  return articlesApi.get(`/articles?topic=${topic}`).then((response) => {
+    return response.data;
+  })
+}
```

Comment:
the code above (fetchArticles) will doe what this fetchArticlesBy topic does! So this is a bit repetitive.